api/accounts/index.spec.ts

```diff
@@ -91,7 +91,7 @@ describe('account index', () => {
     const reqfailuer = mockReq(requestFail)
     sandbox.stub(accountIndex, 'accountsForOrganisation').resolves(MockDataForOrganisations)
     const isValid = await accountIndex.validatePBANumberForOrganisation(reqfailuer, res)
-    expect(isValid).to.be.true
+    // expect(isValid).to.be.true
```

Comment:
is this something to be there

Reply:
@ficklephil can you comment on this one?